### PR TITLE
Added simple navigation menu

### PR DESCRIPTION
**Pull request by HizkiFW**
This PR adds a simple collapsible menu, styled with some animations and support for tab navigation.

### CODE REVIEW

File: openwisp_utils/admin_theme/static/admin/css/openwisp.css [hunk 1]
```diff
@@ -15,6 +15,43 @@
...
 }
 
+
+#nav-activate {
+    cursor: pointer;
+    margin-top: 10px;
+    padding: 0 20px;
+    float: left;
+    background-color: #353535;
+}
```
Comment (nemesisdesign) [inline openwisp_utils/admin_theme/static/admin/css/openwisp.css]: add blank line

File: openwisp_utils/admin_theme/static/admin/css/openwisp.css [hunk 2]
```diff
@@ -15,6 +15,43 @@
...
+#nav-activate:hover {
+    background-color: #222;
+}
+#nav-activate:focus {
```
Comment (nemesisdesign) [inline openwisp_utils/admin_theme/static/admin/css/openwisp.css]: add blank line

File: openwisp_utils/admin_theme/static/admin/css/openwisp.css [hunk 3]
```diff
@@ -15,6 +15,43 @@
...
+    outline: none;
+}
+#nav-top {
```
Comment (nemesisdesign) [inline openwisp_utils/admin_theme/static/admin/css/openwisp.css]: add blank line

File: openwisp_utils/admin_theme/static/admin/css/openwisp.css [hunk 4]
```diff
@@ -15,6 +15,43 @@
...
+    padding: 0;
+    max-height: 0;
+    overflow: hidden;
+    position: absolute;
+    left: 210px;
+    top: 20px;
+    background-color: #353535;
+    transition: all .5s;
+    z-index: 999;
+}
```
Comment (nemesisdesign) [inline openwisp_utils/admin_theme/static/admin/css/openwisp.css]: add blank line

File: openwisp_utils/admin_theme/static/admin/css/openwisp.css [hunk 5]
```diff
@@ -15,6 +15,43 @@
...
+    opacity: 1;
+    max-height: 200%;
+    padding: 1em 2em;
+}
+
+.nav-header {
+    font-weight: bold;
+}
+
+
```
Comment (nemesisdesign) [inline openwisp_utils/admin_theme/static/admin/css/openwisp.css]: remove blank line

**Comment by HizkiFW** [comment]
@nemesisdesign Commits squashed

File: openwisp_utils/admin.py [hunk 6]
```diff
@@ -1,8 +1,8 @@
 from django.contrib import admin
+from django.contrib.admin import *
```
Comment (rohithasrk) [inline openwisp_utils/admin.py]: Redundant. As we've already imported admin from django.contrib

File: openwisp_utils/admin.py [hunk 7]
```diff
@@ -96,3 +96,44 @@ class TimeReadonlyAdminMixin(object):
     def __init__(self, *args, **kwargs):
         self.readonly_fields += ('created', 'modified',)
         super(TimeReadonlyAdminMixin, self).__init__(*args, **kwargs)
+
+class MyAdminSite(AdminSite):
```
Comment (rohithasrk) [inline openwisp_utils/admin.py]: Can make it `admin.AdminSite`

**Pull request closed without merging**